qlib/contrib/model/pytorch_nn.py

```diff
@@ -133,11 +136,14 @@ def __init__(
         )
 
         self._fitted = False
-        self.dnn_model.cuda()
+        
```

Comment:
PEP8: too many blanks

Reply:
@Derek-Wds   It seems that we have run the pep8 checker but the errors are not raised.
![image](https://user-images.githubusercontent.com/465606/95150927-fa076700-07bb-11eb-95fd-5cb9e107d02b.png)

We use [black ](https://github.com/psf/black)with max line width set to 120 as our pep8 checker.